### PR TITLE
Ludlow/test branch

### PR DESCRIPTION
**Pull request by Topher2014**
Added Arch script. 
Added Arch to readme instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,10 @@ On a mac, run this:
 
 `zsh webserver-darwin.sh`
 
-on linux runn this:
+On Arch linux, the best linux, run this:
```
Comment (ChrisForti) [inline README.md:19]: 😂

File: webserver-archlinux.sh [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  if ( [ -d /var/lib/snapd ] && [ -d /snap ] )
+  then
+    sudo ln -s /var/lib/snapd /snap
+  fi
+
+  if ( snap version )
+  then
+    echo "Waiting for processes..." && sleep 15 
+    sudo snap install multipass
+    sudo snap install multipass
```
Comment (ChrisForti) [inline webserver-archlinux.sh]: Why the same command twice?

File: README.md (hunk 1 above)
Comment (stephenlauck) [reply README.md:19]: haha :+1:

File: webserver-archlinux.sh [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  echo "Multipass is installed" 
+else 
+  echo "Installing Multipass..." 
+  if ( [ ! -d snapd ] ) 
+  then 
+    git clone https://aur.archlinux.org/snapd.git
+  else
+    echo "snapd directory already exists"
+    read -p "Do you wish to use existing directory? (yes/no) " yn
+    case $yn in
```
Comment (stephenlauck) [inline webserver-archlinux.sh]: Weeeeeee! Ok, going with some nested complexity!

File: webserver-darwin.sh
Comment (stephenlauck) [inline webserver-darwin.sh:1]: um, why deleted?

**Review by stephenlauck — CHANGES_REQUESTED** [review]
We should make sure we're not deleting the work we have done in the darwin script. Otherwise, this is interesting and ok to merge.

**Review by ChrisForti — APPROVED** [review]
👍🏼

**Review by stephenlauck — APPROVED** [review]

**Pull request merged**